WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -32,9 +32,7 @@ def scramble_program
         "information" => "#{root_url}regulations/scrambles/",
         "download" => "#{root_url}regulations/scrambles/tnoodle/TNoodle-WCA-0.15.0.jar",
       },
-      "allowed" => [
-        "TNoodle-WCA-0.14.0", "TNoodle-WCA-0.15.0"
-      ],
+      "allowed" => ["TNoodle-WCA-0.15.0"],
```

Comment:
Nit: are the `[]` necessary?

Reply:
Yep!
https://github.com/thewca/tnoodle/blob/efb21b3d5d7aa29c14502278f0a604b3014e0c08/tnoodle-ui/src/Layout.js#L96
https://github.com/thewca/tnoodle/blob/6e28fbfe53a7355f7c55056c1d21f954e7846118/webscrambles/src/main/resources/tnoodle_resources/webapps/ROOT/wca/scramblegen.html#L116